src/redis-benchmark.c

```diff
@@ -427,6 +427,7 @@ static void setClusterKeyHashTag(client c) {
      * updateClusterSlotsConfiguration won't actually do anything, since
      * the updated_slots_count array will be already NULL. */
     if (is_updating_slots) updateClusterSlotsConfiguration();
+    node->current_slot_index = rand() % node->slots_count;
     int slot = node->slots[node->current_slot_index];
```

Comment:
Circling back, why can't we just update this line to 
```suggestion
    int slot = node->slots[rand() % node->slots_count];
```

and delete current_slot_index, it doesn't seemed to be used outside of this function. I suppose we can merge this now since we will "break it" for redis 8.